crm_claim_type/README.rst

```diff
@@ -4,7 +4,7 @@
 CRM Claim Types
 ===============
 
-Includes a way to classify claims adding the concept of type.
+Includes a way to classify claims adding the concept of type and with this it allows to configure claim stages depending on claim types.
```

Comment:
@osvalr add the classify claims by default on this module: Customer, Supplier and Other.